src/pages/BitcoinDashboardV2/components/Checkpoint/Checkpoint.tsx

```
@@ -0,0 +1,137 @@
+import {
+  useGetCheckpointData,
+  useGetCheckpointFee,
+  useGetCheckpointQueue,
+  useGetDepositFee,
+  useGetWithdrawalFee
+} from 'pages/BitcoinDashboardV2/hooks';
+import React, { useEffect, useState } from 'react';
+import styles from './Checkpoint.module.scss';
+import { TransactionInput, TransactionOutput } from './Transactions';
+
+const Checkpoint: React.FC<{}> = ({}) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)